qcommon/src/org/qcmg/common/string/StringUtils.java

```diff
@@ -445,4 +445,14 @@ public static String getStringFromCharSet(Set<Character> set) {
 		return sb.toString();
 	}
 
+	
+	public static <T> String joinByComma( List<T> possibles){		 		
```

Comment:
tests please
also, why not use streams rather than building a string and then having to perform a substring?

Reply:
this method was used a long time ago by qprofiler, but it is no longer required. I deleted this method and will keep in mind to use a new feature from late java version.